app/matplotlib.ipynb

```diff
@@ -306,23 +355,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
-   "outputs": [],
    "source": [
     "x = np.linspace(-5, 5, 100)\n",
     "y = np.linspace(-5, 5, 100)\n",
     "X, Y = np.meshgrid(x, y)\n",
-    "Z = X**2 + Y**2\n",
+    "Z = X ** 2 + Y ** 2\n",
     "\n",
-    "# write your code here\n"
+    "# write your code here"
```

Comment:
The code for creating a 3D surface plot is missing. You need to add the necessary code to create a 3D plot using `Axes3D` and `plot_surface` methods.